src/ucp/wireup/wireup_cm.c

```diff
@@ -492,7 +492,8 @@ static void ucp_ep_cm_disconnect_flushed_cb(ucp_request_t *req)
 
 static unsigned ucp_ep_cm_remote_disconnect_progress(void *arg)
 {
-    ucp_ep_h ucp_ep = arg;
+    static const ucp_request_param_t param = {.op_attr_mask = 0};
```

Comment:
maybe define such empty params on API level, so will not need to modify later?
@yosefe, wdyt?

Reply:
could be separate PR for empty params API

Reply:
what about initializer for empty param?

Reply:
maybe define it as internal in this PR, and expose in another PR
`const ucp_request_param_t ucp_request_null_param = { .op_attr_mask = 0 };`

Reply:
added